readme.md

```diff
@@ -442,8 +442,25 @@ devtools takes the store function as its first argument, optionally you can name
 Name store: `devtools(store, {name: "MyStore"})`, which will create a seperate instance named "MyStore" in the devtools.
 Serialize options: `devtools(store, { serialize: { options: true } })`.  
   
+#### Logging Actions
+
 devtools will only log actions from each separated store unlike in a typical *combined reducers* redux store. See an approach to combining stores https://github.com/pmndrs/zustand/issues/163
 
+Because of this, actions will be logged as "anonymous" by default. To set a default log action type for all actions from your store: `devtools(store, {name: 'MyStore', anonymousActionType: 'MyStoreAction'})`
```

Comment:
The recommended approach is to use the third parameter instead of `anonymousActionType` so let's mention that first.

And then after that let's just write "If the third parameter is not passed, the action type by default becomes `"anonymous"`. You can customize this via the `anonymousActionType` devtools option. Eg `devtool(..., { anonymousActionType: 'unknown', ... })`"

It doesn't make sense to use or encourage the store name as `anonymousActionType` because you can already see the store instance in top right drop-down.